requirements.txt

```
@@ -1,8 +1,5 @@
-numpy>=1.19.2
-scipy>=1.2.1
-numba>=0.48.0
-pytest>=5.4.1
-pytest-randomly>=3.5.0
-flaky>=3.7.0
-sympy>=1.5.1
-dask[delayed]
+numpy==1.21.5
+scipy==1.8.0
+numba==0.55.1
+sympy==1.10
+dask[delayed]==2022.2.1
```

We should keep this list sorted!
```suggestion
dask[delayed]==2022.2.1
numba==0.55.1
numpy==1.21.5
scipy==1.8.0
sympy==1.10
```